### ISSUE TITLE
Extension chrome

### ISSUE THREAD

**Issue by DOCPROF** [jmoenig/Snap--Build-Your-Own-Blocks#1273]
An idea : Create a Chrome extension offline to use snap! on chromebook.

**Comment by cycomachead** [comment]
Various forms of extensions and apps are on the list of ideas. 😄 We will probably get to it someday, but they're not quite a priority right now...

There are other discussions about improving general offline access that we're talking about, which might happen more quickly. (Basically, we just need to tell browsers how to download all of Snap!'s resources.)

**Comment by jmoenig** [comment]
I think Tom Lauwers may have already done just that, and perhaps @bromagosa , too ... (?)

**Comment by dekrain** [comment]
I have an idea.
Add a file like "snap_files.yml" .yml is example and add browsers to find this file (snap.berkeley.edu /snap_files or /snapsource/[dev/]snap_files) and download other files

**Comment by dekrain** [comment]
Or .json

**Comment by bromagosa** [comment]
Yep, I've done that in the past with Snap4Arduino: http://scaenaweb.citilab.eu/snap4arduino/Snap4Arduino-chrome.crx

It'd be really easy to do it for just Snap!, as it's a client application :)
